### PR TITLE
[SPARK-35733][SQL][TESTS] Check all day-time interval types in `HiveInspectors` tests

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### What changes were proposed in this pull request?
Check all day-time interval types in HiveInspectors tests.

### Why are the changes needed?
New tests should improve test coverage for day-time interval types.

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
Added UT.

### CODE REVIEW

**Comment by AngersZhuuuu** [comment]
FYI @MaxGekk

File: sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveScriptTransformationSuite.scala [hunk 1]
```diff
@@ -522,32 +523,53 @@ class HiveScriptTransformationSuite extends BaseScriptTransformationSuite with T
 
   }
 
-  test("SPARK-34879: HiveInspectors supports DayTimeIntervalType") {
+  test("SPARK-35733: HiveInspectors supports supports all type of DayTimeIntervalType") {
```
Comment (MaxGekk) [inline sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveScriptTransformationSuite.scala:526]: Please, add both: `test("SPARK-34879, SPARK-35733:`

Comment (AngersZhuuuu) [reply sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveScriptTransformationSuite.scala]: Done

File: sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveScriptTransformationSuite.scala [hunk 2]
```diff
@@ -522,36 +523,57 @@ class HiveScriptTransformationSuite extends BaseScriptTransformationSuite with T
...
+            AttributeReference("j", DayTimeIntervalType(SECOND))()),
           child = child,
           ioschema = hiveIOSchema),
-        df.select($"a", $"b", $"c").collect())
+        df.select($"a", $"b", $"c", $"d", $"e", $"f", $"g", $"h", $"i", $"j").collect())
     }
   }
 
-  test("SPARK-35722: HiveInspectors supports all type of YearMonthIntervalType") {
+  test("SPARK-34879, SPARK-35722: HiveInspectors supports all type of YearMonthIntervalType") {
```
Comment (AngersZhuuuu) [inline sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveScriptTransformationSuite.scala:576]: @MaxGekk  Also update here

**Review by MaxGekk — APPROVED** [review]
+1, LGTM. Merging to master.
Thank you, @AngersZhuuuu .

**Pull request closed without merging**